### DISCUSSION TITLE
How to install and use Paste in an existing project without breaking Flex UI?

### DISCUSSION THREAD

**Discussion by WillCKyle** [twilio-labs/paste#2738] [Q&A] [answered]
I manually installed Paste in the base directory of my typescript-based plugin by following the documentation here: [Paste Manual Installation - Paste: The Design System for building Twilio customer experiences ](https://paste.twilio.design/introduction/for-engineers/manual-installation/) 

This seemed to cause problems for Twilio Flex UI. Before this installation, the plugin could successfully place MUI components within a tab created within TaskCanvasTabs, following this documentation: [Add, Replace, and Remove Flex UI Components | Twilio](https://www.twilio.com/docs/flex/developer/ui/components#taskcanvastabs)  

After installation, "Tab" now returns an error in visual studio code, saying "'Tab' cannot be used as a JSX component." Now I cannot get any content, Paste or otherwise, to appear in any area of the Flex UI. The plugin compiles in a local test environment without reporting errors, but nothing from the plugin appears. This leads me to think that the Flex UI commands (e.g. Flex.AgentDesktopView.Panel2.Content.add()) is no longer working.

**Comment by TheSisb** [comment, marked as answer]
Hi @WillCKyle,

Unfortunately this question is a little too nuanced about the Flex API for our team to answer. As per [my other comment in your other thread](https://github.com/twilio-labs/paste/discussions/2742), I would highly advise to check in with someone from the Flex support side for an answer.
